src/components/Error/Error.tsx

```diff
@@ -1,11 +1,18 @@
 import React, { useState, useEffect } from 'react';
 
-
 const Error: React.FC = () => {
 
   return (
-
+    <div>
+      <h1>404error</h1>
+      <div>
+        Menu
+      </div>
```

Comment:
noise!

Reply:
Are we importing the same CSS everywhere or just for specific files?

Reply:
Ideally everywhere.